bundles/org.openhab.binding.nikobus/README.md

```diff
@@ -196,140 +172,6 @@ Thing push-button 28092A [ impactedModules = "4C6C-2" ]
-
-in `nikobus.cfg` than add
-
-```
-Bridge nikobus:pc-link:mypclink [ port = "/dev/ttyUSB0", refreshInterval = 30 ] {
-}
-```
-
-In order to use Nikobus modules, each needs to be defined within the `.things` file:
-
-* using `switch-module` for switching module i.e. 05-000-02,
-* using `dimmer-module` for dim-controller i.e 05-007-02,
-* using `rollershutter-module` for rollershutter module.
-
-Address of the modules are the same as they were in v1 binding.
-
-So during v1 -> v2 migration one needs to go through the v1 `.items` file and for each line like:
-
-```
-Switch Light_GF_Kitchen_Bar "Bar" (GF_Kitchen, Lights) { nikobus="4C6C:3" }
-```
-
-add to the `.things` file (if not added already):
-
-```
-Thing switch-module 4C6C
-```
-
-for each line like:
-
-```
-Dimmer Light_GF_Dining_Ceiling "Ceiling" (GF_Dining, Lights) { nikobus="6B00:10" }
-```
-
-add
-
-```
-Thing dimmer-module 6B00
-```
-
-for each line like:
-
-```
-Rollershutter Shutter_GF_Corridor "Corridor" (GF_Corridor, gShuttersGF) { nikobus="D969:1" }
-```
-
-add
-
-```
-Thing rollershutter-module D969
-```
-
-and for each line like:
-
-```
-Switch S_1_15_2A { nikobus="#N9D592A[4C6C-1]" }
-```
-
-add
-
-```
-Thing push-button 9D592A "S_1_15_2A" [ impactedModules = "4C6C-1" ]
-```
-
-Make sure the `#N` is removed. Also, please note if such items (push buttons) are not used in the `.sitemap`, they should be removed from the `.items` file.
-
-At the end the `.things` file should look something like:
-
-```
-Bridge nikobus:pc-link:mypclink [ port = "/dev/ttyUSB0", refreshInterval = 30 ] {
-
-    Thing dimmer-module 6B00
-    Thing dimmer-module 0700
-
-    Thing switch-module BC00
-    Thing switch-module 4C6C
-    Thing switch-module A063
-
-    Thing rollershutter-module D969
-
-    Thing push-button 9D592A "S_1_15_2A" [ impactedModules = "4C6C-1" ]
-}
-```
-
-Please make sure the mapping is correct - switch to `switch-module`, dim-controller to `dimmer-module` and rollershutter controller to `rollershutter-module`.
-
-Next step is to migrate `.items` file. Entries as:
-
-```
-Switch Light_GF_Dining_Wall "Wall" (GF_Dining, Lights) [ "Lighting" ] { nikobus="4C6C:12" }
-```
-
-should be updated to
-
-```
-Switch Light_GF_Dining_Wall "Wall" (GF_Dining, Lights) [ "Lighting" ] { nikobus="nikobus:switch-module:mypclink:4C6C:output-12" }
-```
-
-entries as
-
-```
-Dimmer Light_GF_Dining_Ceiling 	"Ceiling" (GF_Dining, Lights) { nikobus="6B00:10" }
-```
-
-should be updated to
-
-```
-Dimmer Light_GF_Dining_Ceiling 	"Ceiling" (GF_Dining, Lights) { channel="nikobus:dimmer-module:mypclink:6B00:output-10" }
-```
-
-and entries as
-
-```
-Rollershutter Shutter_GF_Corridor "Corridor" (GF_Corridor, gShuttersGF) {nikobus="D969:1"}
-```
-
-should be updated to
-
-```
-Rollershutter Shutter_GF_Corridor "Corridor" (GF_Corridor, gShuttersGF) {nikobus="nikobus:rollershutter-module:mypclink:D969:output-1"}
-```
+## Full Example
 
-With above in place, existing `.sitemap` should work as before.
+t.b.d.
```

Comment:
@crnjan Could you please add this "Full Example" section, which is mandatory for all bindings? It should show an example setup with a thing/item/sitemap file, which makes it clear, how those fit together.